spread.yaml

```diff
@@ -42,7 +42,7 @@ environment:
     NEW_CORE_CHANNEL: "$(HOST: echo $SPREAD_NEW_CORE_CHANNEL)"
     SRU_VALIDATION: "$(HOST: echo ${SPREAD_SRU_VALIDATION:-0})"
     PRE_CACHE_SNAPS: core ubuntu-core test-snapd-tools
-    SKIP_REMOVE_SNAPS: "$(HOST: echo ${SKIP_REMOVE_SNAPS:-})"
+    SKIP_REMOVE_SNAPS: "$(HOST: echo ${SPREAD_SKIP_REMOVE_SNAPS:-}) rsync"
```

Comment:
It took me a bit to figure out what this is used for, maybe a small comment like: `# always skip removing the rsync snap, its essential in prepare/restore' ?

Reply:
SPREAD_SKIP_REMOVE_SNAPS is just used on ubuntu core, and it is needed for preparing tests. We need it because we don't have rsync command on ubuntu core.